tasks/sep/032-op-pectra-defence/.env

```diff
@@ -0,0 +1,9 @@
+ETH_RPC_URL="https://ethereum-sepolia.publicnode.com"
+
+USE_MULTI_DELEGATECALL=true
+TENDERLY_GAS=20000000
+
+SAFE_NONCE=""
```

Comment:
I think we'll end up using the following nonces (or +1 of these nonces) because I plan to execute an Ink Sepolia's upgrade tomorrow and have another queued up.

```
SAFE_NONCE_0X1EB2FFC903729A0F03966B917003800B145F56E2=20
SAFE_NONCE_0XF64BC17485F0B4EA5F06A96514182FC4CB561977=29
SAFE_NONCE_0XDEE57160AAFCF04C34C887B5962D0A69676D3C8B=34
```

Reply:
ok, thanks for the heads up!

Reply:
Leaving this comment open until we are certain about the nonces.

Reply:
I've update the .env file with the current nonces which are now:

```
SAFE_NONCE_0X1EB2FFC903729A0F03966B917003800B145F56E2=21
SAFE_NONCE_0XF64BC17485F0B4EA5F06A96514182FC4CB561977=30
SAFE_NONCE_0XDEE57160AAFCF04C34C887B5962D0A69676D3C8B=35
```

Reply:
I'm told that there are no tasks between now and when this will execute.